source/helpers.js

```diff
@@ -15,3 +36,123 @@ exports.generateTextBar = function (numerator, denominator, barLength) {
 	}
 	return bar;
 }
+
+exports.getNumberEmoji = function (number) {
+	switch (Number(number)) {
+		case 0:
+			return '0️⃣';
+		case 1:
+			return '1️⃣';
+		case 2:
+			return '2️⃣';
+		case 3:
+			return '3️⃣';
+		case 4:
+			return '4️⃣';
+		case 5:
+			return '5️⃣';
+		case 6:
+			return '6️⃣';
+		case 7:
+			return '7️⃣';
+		case 8:
+			return '8️⃣';
+		case 9:
+			return '9️⃣';
+		case 10:
+			return '🔟';
+		default:
+			return '#️⃣';
+	}
+}
```

Comment:
As much as this looks somewhat textbook for a switch case block, it also feels...inefficient? the 0 through 10 case could be pulled from a constant array that has each of those at the corresponding index, and that somewhat reduces this into an if...else (if num >=0 && numb <= 10 ...).

Reply:
Probably better as a hash table tbh

Reply:
Done in https://github.com/Imaginary-Horizons-Productions/BountyBot/pull/23/commits/0797fc93322413eb711e7ac461e854b3810b2cda